main/deleteCandidates.html

```diff
@@ -41,6 +41,10 @@ <h1>Mass Candidate Deletion Tool</h1>
         <textarea id="candidateIds" rows="4"></textarea>
     </div>
     <button onclick="startDeleteCandidates()">Delete Candidates</button>
+    <div id="output-progress" class="flex-left-right-container" style="visibility:hidden">
+        <p>Requests Sent: <span id="count">0</span></p> 
+        <button id="stop" class="paradox-primary-btn grey-text text-lighten-5" onclick="stopRequest()">Stop</button>
+    </div>
```

Comment:
Html element to display the number of requests and the stop button.

Reply:
Does Visibility:hidden still consume space in the document vs display:none? Also, Materialize has a class named hide that can be toggled vs js toggle